tests/orm/test_querybuilder.py

```diff
@@ -740,7 +740,11 @@ def test_queryhelp(self):
         self.assertEqual(qb.count(), 1)
 
     def test_recreate_from_queryhelp(self):
-        """Test recreating a QueryBuilder from the Query Help"""
+        """
+        Test recreating a QueryBuilder from the Query Help
```

Comment:
```suggestion
        """Test recreating a QueryBuilder from the Query Help.
        
```

Reply:
Ok, I will change this, but I thought that the convention was to use

````
"""Single line header"""

---

"""
Multiline
Header
"""
````

Did I misunderstand this?

Reply:
Yes, the first line of a docstring should always stay on a single line, be a single short description in the imperative mood and end with a period. If just a single line, the closing quotes are also on the same line. For multiline string, an empty line is created after the first subject line and the closing quotes are on a separate line. Examples:
```
"""Do something with the given inputs."""
```
Multiline
```
"""Do something with the given inputs.

This function does something particular that you may not expect.
"""
```
These rules are following the [PEP 257](https://www.python.org/dev/peps/pep-0257/) conventions.